conans/client/conan_command_output.py

```diff
@@ -151,13 +153,21 @@ def _grab_info_data(self, deps_graph, grab_paths):
                 item_data["package_folder"] = package_layout.package(pref)
 
             try:
-                reg_remote = self._cache.package_layout(ref).load_metadata().recipe.remote
+                package_metadata = self._cache.package_layout(ref).load_metadata()
+                reg_remote = package_metadata.recipe.remote
                 reg_remote = remotes.get(reg_remote)
                 if reg_remote:
                     item_data["remote"] = {"name": reg_remote.name, "url": reg_remote.url}
             except Exception:
                 pass
 
+            try:
+                package_revision = package_metadata.packages[package_id].revision
```

Comment:
This will fail if the above try-except fails, and it will fail in an ugly way, but will be hidden.
I'd say better add this in the above try-except, below the ``if reg_remote:``